### DISCUSSION TITLE
How to display widgets on the whole page of the browser?

### DISCUSSION THREAD

**Discussion by hassan-anari** [rio-labs/rio#180] [Q&A] [answered]
Hello, the question may be very simple and small, but it really is a question for me!
For example, look at the image below, this is an example of the project, but it is not displayed in the whole browser page and it is being displayed in the second half...
![image](https://github.com/user-attachments/assets/df6b3fed-ead0-4c1e-ac79-c4bb34c1417e)
What should I do? thank you.

**Comment by Aran-Fey** [comment]
I think it's probably best if we take a look at the generated HTML code for the app. Please open the app in your browser, then save the HTML (via `Ctrl+S` or the `File -> Save As` menu) and upload it here.

**Comment by hassan-anari** [comment]
Sure, Thanks.
[data.zip](https://github.com/user-attachments/files/17902535/data.zip)

**Comment by Aran-Fey** [comment, marked as answer]
It seems like your VeePN browser extension is inserting an element into the body, which is what screws up the layout. Not entirely sure how to deal with this, to be honest. We *could* just set foreign elements to `display: none`, but that probably wouldn't be very wise...

**Comment by mad-moo** [comment]
While that wouldn't be a perfect solution, it's still much preferable than the alternative. Better breaka _buggy_ browser extension, than have them break websites. This is now the third (?) time we've seen browser extensions interfere with layouts

**Reply by hassan-anari** [reply]
Undoubtedly, in the future, solutions will be found for these issues and these problems will also be solved... 
Thank you for your guidance.
